src/qml/bitcoin.cpp

```diff
@@ -93,7 +95,7 @@ AppMode SetupAppMode()
         wallet_enabled = false;
     #endif // ENABLE_WALLET
 
-    return AppMode(mode, wallet_enabled);
+    return new AppMode(mode, wallet_enabled);
```

Comment:
Why this change is required? Is it safe?

Reply:
Looking into it...

Reply:
Thanks for catching that, no it is not safe! While refactoring ran into some compiling errors and thought (mistakenly) that turning `AppMode` into a pointer would be the way to go. That’s not necessary so reverting back to the original definition.